wire-library/wire-compiler/src/main/java/com/squareup/wire/schema/Target.kt

```diff
@@ -255,7 +255,7 @@ data class JavaTarget(
             javaFile.packageName.replace(".", "/") /
             "${javaTypeName.simpleName()}.java"
 
-        logger.artifactHandled(modulePath, "${javaFile.packageName}.${javaFile.typeSpec.name}")
+        logger.artifactHandled(modulePath, "${javaFile.packageName}.${javaFile.typeSpec.name}", "Java")
```

Comment:
Could we pass the two strings into `artifactHandled` as `packageName` and `typeName` and let the logging close itself handle the interpolation / output?

Reply:
How about proto generation? And swift ( `"${swiftFile.moduleName}.${typeName.canonicalName}",` ) ?

My main thinking is something like:

- Wire has some built in targets (Java, Kotlin, Proto, Swift).
- Here is what to expect from Wire's built-in targets' logs.
- But Wire doesn't wanna force an API that would require gymnastic or awkward naming from its consumers.

Broadly, if something is generated, there's a location where it was generated, and its id (for java that's qualified name, for protos just the file name, etc)

It could be good to renamed `artifactHandled` to `artifactGenerated` and another `artifactProcessed` or something for side effect which don't generate code.